todo-list/components/layout.js

```diff
@@ -0,0 +1,69 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import styles from './layout.module.css';
+import utilStyles from '../styles/utils.module.css';
+import Link from 'next/link';
+
+const name = 'Vasyl Luchka';
+export const siteTitle = 'Next.js Sample Website';
```

Comment:
Краще винести ці константи в загальні константи в файл `configs/index.ts`.

Також їх глобальні константи рекомендуємо називати через `CAPITAL_SNAKE_CASE`

```suggestion
const NAME = 'Vasyl Luchka';
export const DOCUMENT_TITLE = 'Next.js Sample Website';
```